doc/modules/changes.h

```diff
@@ -4,6 +4,13 @@
  * <p> This is the list of changes made after the release of Aspect version
  * 1.4.0. All entries are signed with the names of the author. </p>
  *
+ * <li> New: A "bound preserving limiter" working as a postprocessor is applied 
```

Comment:
Please reformulate this a bit. It should be clear that this is a new *optional* feature. Additionally, we use "postprocessor" mostly in the sense that it does not change the solution, but your limiter does. You probably mean "a correction after solving the advection equation".